### PR TITLE
English Translation

### PR DESCRIPTION
**Pull request by brucemeek**
I have translated all the Chinese to English

### CODE REVIEW

File: metagpt/config.py [hunk 1]
```diff
@@ -40,7 +40,7 @@ class Config(metaclass=Singleton):
 
     def __init__(self, yaml_file=default_yaml_file):
         self._configs = {}
-        self._init_with_config_files_and_env(self._configs, yaml_file)
+        self._initialize_with_config_files_and_environment(self._configs, yaml_file)
```
Comment (geekan) [inline metagpt/config.py:43]: too long method.

File: metagpt/tools/ut_writer.py [hunk 2]
```diff
@@ -6,178 +6,177 @@
...
+Method: POST
+
+Request Parameters:
+Path Parameters:
+
+Body Parameters:
+Name        Type       Required    Default Value    Remarks
+file_id     string     Yes         
+box         array      Yes         
+contract_id number     Yes         Contract ID
```
Comment (geekan) [inline metagpt/tools/ut_writer.py:86]: dont change tab to spaces

**Review by geekan — COMMENTED** [review]
Thank you so much! Have you verified the correctness of the results? The modification of prompts may cause changes in the results. The UT generation part here has actually been carefully designed, but I am not sure if it has been verified?

Besides that, there should be some Chinese. I don't know how you found out if there is any Chinese left?

File: metagpt/utils/common.py [hunk 3]
```diff
@@ -213,14 +214,4 @@ def print_members(module, indent=0):
     prefix = ' ' * indent
     for name, obj in inspect.getmembers(module):
         print(name, obj)
-        if inspect.isclass(obj):
```
Comment (geekan) [inline metagpt/utils/common.py:223]: is it correct?

Comment (Kaplarr) [reply metagpt/utils/common.py:223]: So I wonder too. Is it?

**Comment by geekan** [comment]
@brucemeek could you solve the conflict?

**Comment by brucemeek** [comment]
> @brucemeek could you solve the conflict?

I am doing another clean pass of your original files, when translating it, It seems some minor things were left out. Ill update mine once i finish the translation. Also what parts need to stay chinese??

**Comment by brucemeek** [comment]
can someone please merge this now. All conflicts are fixed.

**Comment by ImJohnDoe** [comment]
This is very important, I would love more focus on merging this PR soon.

**Comment by stellaHSR** [comment]
LGTM

**Pull request merged**